views/timeslot.tmpl

```diff
@@ -17,20 +17,24 @@
 {{end}}
 
 {{define "content"}}
-{{with .PageData}}
 <div class="container-fluid current-next">
   <div class="container">
-  {{if .MixcloudAvailable}}
+  {{if .PageData.MixcloudAvailable}}
     <div class="row justify-content-center">
       <div class="col-11 col-md-10 py-2 mixcloud">
-        <h2>URY on Tap: Listen Again</h2>
-        <iframe width="100%" height="120" src="https://www.mixcloud.com/widget/iframe/?feed={{.Timeslot.MixcloudStatus}}&hide_cover=1&light=1" frameborder="0"></iframe>
-        <a href="#">Listen to more shows on URY On Demand</a>
+        <h2>
+          <a href="/ontap/" title="Goto {{.PageContext.ShortName}} {{.PageContext.ODName}}">
+            {{- .PageContext.ShortName}} {{.PageContext.ODName -}}
+          </a>: Listen Again
+        </h2>
+        <iframe width="100%" height="120" src="https://www.mixcloud.com/widget/iframe/?feed={{.PageData.Timeslot.MixcloudStatus}}&hide_cover=1&light=1" frameborder="0"></iframe>
```

Comment:
hardcoded protocol

Reply:
whats the disadvantage to this? mix cloud will only load over https?